tests/testthat/test-sce-subset.R

```diff
@@ -1,22 +1,28 @@
 # Checks the subsetting methods.
 # library(SingleCellExperiment); library(testthat); source("test-sce-subset.R")
+context("SCE subsetting")
 
 set.seed(1000)
 ncells <- 100
 v <- matrix(rnorm(20000), ncol=ncells)
 pca <- matrix(runif(ncells*5), ncells)
-sce <- SingleCellExperiment(assay=v, reducedDims=SimpleList(PCA=pca)) # A fully loaded object.
```

Comment:
Duplicate the test rather than replacing it with an LEM example.

Reply:
Wait, I see what you've done. In that case, duplicate the test in the new test file (`test-sce-with-lem.R`) that you've created. Isolate LEM+SCE tests from the SCE-only tests to a single file.

Reply:
Also test `cbind`. This is actually quite complex as it only makes sense to `cbind` LEMs that have the same loading vectors; should probably check that in `cbind,LEM-method`.